__tests__/InlineHtmlStripStylesTransformer.test.ts

```diff
@@ -0,0 +1,214 @@
+/*
+ * Code is inspired by
+ * https://github.com/kulshekhar/ts-jest/blob/25e1c63dd3797793b0f46fa52fdee580b46f66ae/src/transformers/hoist-jest.spec.ts
+ * original license:
+ * 
+ * MIT License
+ * 
+ * Copyright (c) 2016-2018
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ * 
+ */
+
+import * as tsc from 'typescript'
+import * as transformer from '../InlineHtmlStripStylesTransformer'
+
+const CODE_WITH_TEMPLATE_URL = `
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: './page.html'
+})
+export class AngularComponent {
+}
+`
+
+const CODE_WITH_STYLE_URLS = `
+import { Component } from '@angular/core';
+
+@Component({
+  styleUrls: [
+    './fancy-styles.css',
+    './basic-styles.scss'
+  ]
+})
+export class AngularComponent {
+}
+`
+
+const CODE_WITH_STYLES = `
+import { Component } from '@angular/core';
+
+@Component({
+  styles: [
+    'body: { display: none }',
+    'html: { background-color: red }'
+  ]
+})
+export class AngularComponent {
+}
+`
+
+const CODE_WITH_ALL_DECORATOR_PROPERTIES = `
+import { Component } from '@angular/core';
```

Comment:
some indentation wouldn't hurt (and will be easy to adjust once output tests are converted to snapshots